File: Arbitration/MPArbitration/Model/AuthorityDisputeWorkItem.cs
using System.Text.Json.Serialization;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;


namespace MPArbitration.Model
{
    
    public class AuthorityDisputeWorkItem
    {
        [JsonPropertyName("assignedUser")]
        public string AssignedUser { get; set; } = "";

        [JsonPropertyName("disputeId")]
        public int DisputeId { get; set; } = 0;

        [JsonPropertyName("note")]
        public AuthorityDisputeNote? Note { get; set; } = null;

        [JsonPropertyName("workQueue")]
        public WorkQueueName WorkQueue { get; set; } = WorkQueueName.None;
    }
    
}

File: Arbitration/MPArbitration/ClientApp/src/app/model/authority-dispute-work-item.ts
import { LogLevel } from "@azure/msal-browser";
import { loggerCallback } from "../app.module";
import { AuthorityDisputeNote } from "./note";
import { WorkQueueName } from "./work-queue-name-enum";

export class AuthorityDisputeWorkItem
{
    assignedUser = '';
    disputeId = 0;
    note:AuthorityDisputeNote|undefined;
    workQueue:WorkQueueName = WorkQueueName.None;

    constructor(obj?:any) {
        if(!obj)
            return;

        Object.assign(this, JSON.parse(JSON.stringify(obj)));
        try {
            if(!!this.note) {
                this.note = new AuthorityDisputeNote(this.note);
            }
        } catch(err) {
            loggerCallback(LogLevel.Error, 'Unable to instantiate Entity');
            console.error(err);
        }
    }
}